src/components/ItemEditorPage/CenterPanel/CenterPanel.tsx

```diff
@@ -23,39 +37,161 @@ export default class CenterPanel extends React.PureComponent<Props> {
-                  value={avatarAnimation}
-                  options={Object.values(AvatarAnimation).map(value => ({ value, text: t(`avatar_animations.${value}`) }))}
-                  onChange={this.handleAnimationChange}
-                />
-              </div>
-            }
-          />
+        <div className="footer">
+          <div className="options">
+            <div className={`option ${isShowingAvatarAttributes ? 'active' : ''}`} onClick={this.handleToggleShowingAvatarAttributes}>
+              <Icon name="user" />
+            </div>
+            <Popup
+              className="disabled-animation-tooltip"
+              content={t('item_editor.center_panel.disabled_animation_dropdown')}
+              disabled
+              position="top center"
+              trigger={
+                <div className="avatar-animation-dropdown-wrapper option">
+                  <Dropdown
+                    className="avatar-animation"
+                    value={avatarAnimation}
+                    options={Object.values(AvatarAnimation).map(value => ({ value, text: t(`avatar_animations.${value}`) }))}
+                    onChange={this.handleAnimationChange}
+                  />
+                </div>
+              }
+            />
+          </div>
+          <div className={`avatar-attributes ${isShowingAvatarAttributes ? 'active' : ''}`}>
+            <div className="dropdown-container">
+              <Dropdown
+                inline
+                direction="right"
+                className="Select"
+                onChange={this.handleBodyShapeChange}
+                {...this.getDropdownAttributes(
+                  bodyShape,
+                  [
+                    { value: WearableBodyShape.MALE, text: t('body_shapes.male') },
+                    { value: WearableBodyShape.FEMALE, text: t('body_shapes.female') }
+                  ],
+                  t('wearable.category.body_shape')
+                )}
+              />
+            </div>
+
+            <AvatarColorDropdown
+              currentTone={skinColor}
+              tones={getSkinColors()}
+              label={t('wearable.color.skin')}
+              onChange={this.handleSkinColorChange}
+            />
+            <AvatarColorDropdown
+              currentTone={eyeColor}
+              tones={getEyeColors()}
+              label={t('wearable.color.eye')}
+              onChange={this.handleEyeColorChange}
+            />
+            <AvatarColorDropdown
+              currentTone={hairColor}
+              tones={getHairColors()}
+              label={t('wearable.color.hair')}
+              onChange={this.handleHairColorChange}
+            />
+          </div>
         </div>
       </div>
     )
   }
 }
+
+type AvatarColorDropdownProps = {
+  currentTone: Color4
+  tones: Color4[]
+  label: string
+  onChange: (color: Color4) => void
+}
+export class AvatarColorDropdown extends React.PureComponent<AvatarColorDropdownProps> {
```

Comment:
What do you think of moving this component to a new file?. Could be hard to find if we need to reuse it.

Reply:
Actualy skipped unintentionally, was the intention!